### PR TITLE
added balaji assignment

### PR DESCRIPTION
**Pull request by jamge**

### CODE REVIEW

**Review by jarandeshubham — APPROVED** [review]

File: balaji/html1.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+<title> </title>
+</head>
+
+<body>
+    <h1>HTML </h1>
+<h2>introduction</h2>
+    <h6> HTML is standard markup language for web page</h6>
+</body>
+</html>
```
Comment (Pawan1005) [inline balaji/html1.html:14]: add new line at the end of file

File: balaji/html2.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+<body> 
+    <h1>HTML</h1>
+    <h2>introduction</h2>
+    <p> HTML is standard markup language for web pages.</p>
+    <h2> HTML editor</h2>
+    <h4> Notepad or TextEdit.</h4>
+    <p> web pages can be created and modified by using professional HTML editor.<br>
+    Howeever,for learning HTML we recommend a simple text editor like NOtpad (PC) or TextEdit(Mac).</p>
+</body>
+
```
Comment (Pawan1005) [inline balaji/html2.html:15]: remove extra lines

File: balaji/html2.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+
+
+
+
+</html>
```
Comment (Pawan1005) [inline balaji/html2.html:21]: add new line

File: balaji/html3.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    <hr>
+    <h2> HTML editor</h2>
+    <h4> Notepad or TextEdit.</h4>
+    <p> web pages can be created and modified by using professional HTML editor.<br>
+    Howeever,for learning HTML we recommend a simple text editor like NOtpad (PC) or TextEdit(Mac).</p>
+</body>
+
+
+
+
```
Comment (Pawan1005) [inline balaji/html3.html:19]: remove extra lines

File: balaji/html3.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title></title>
```
Comment (Pawan1005) [inline balaji/html3.html:4]: add title

File: balaji/html3.html [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    <p> web pages can be created and modified by using professional HTML editor.<br>
+    Howeever,for learning HTML we recommend a simple text editor like NOtpad (PC) or TextEdit(Mac).</p>
+</body>
+
+
+
+
+
+
+</html>
```
Comment (Pawan1005) [inline balaji/html3.html:22]: add new line

File: balaji/html4.html [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+        </title>
+    </head>
+    <body>
+        <h1>I Am Balaji Vitthalrao Jamge <h1>
+
+        <h2>I Am Basically Nanded but Currently Staying in Pune, 
+I Did My Post-Graduation Completed in M.Sc. Computer Science from Swami Ramanand Teertha Marathwada University Nanded
+        </h2>
+    </body>
+</html>
```
Comment (Pawan1005) [inline balaji/html4.html:15]: add new line

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**